### ISSUE TITLE
[Proposal] Is it better use first Error message in ValidateException instead of dead code "The given data was invalid"? 

### ISSUE THREAD

**Issue by jianfengye** [laravel/internals#947]
- Laravel Version: 5.5
- PHP Version:7.2
- Database Driver & Version:

### Description:
Is it better use first Error message in ValidateException instead of dead code "The given data was invalid";

In VlidateException.php， there is code:
```
public function __construct($validator, $response = null, $errorBag = 'default')
    {
        parent::__construct('The given data was invalid.');

        $this->response = $response;
        $this->errorBag = $errorBag;
        $this->validator = $validator;
    }
```
if i use that code in controller:
```
Validator::make($request->all(), [
            'phone' => 'required|unique:users|numeric',
            'code' => 'required|numeric',
        ],[
            'phone.unique' => "the user has been registered"
        ])->validate();
```
and the api will response exception:
```
{message: "The given data was invalid"}
```

Is it better to set Exception's message to $validator->errors()->first()?

because first error message is meanful and can be custom or localize, but "The given data was invalid" has no meaning

### Steps To Reproduce:

NONE

**Comment by mikerockett** [comment]
I don’t agree on the part about the first error being used. That `message` should (imo) be used for displaying a generalised message to prompt the user to check their input.

What I do agree on, however, is that this should be customisable through a `message()` (or something like that) method in a form request, or another argument in the `validate` and `make` methods.

You could always [override it](https://blog.rockett.pw/post/how-to-add-a-custom-validation-message-in-laravel-5.5-form-requests) to suit your needs until this is made available in the core (I’m hopeful that it’ll happen).
